src/masternodes/govvariables/attributes.cpp

```diff
@@ -344,6 +364,60 @@ static bool VerifyToken(const CCustomCSView& view, const uint32_t id) {
     return view.GetToken(DCT_ID{id}).has_value();
 }
 
+static ResVal<CAttributeValue> VerifyConsortiumMember(const std::string& str) {
+    UniValue values(UniValue::VOBJ);
+    CConsortiumMembers members;
+
+    if (!values.read(str))
+        return Res::Err("Not a valid consortium member object!");
+
+    for (const auto &key : values.getKeys())
+    {
+        UniValue value(values[key].get_obj());
+        CConsortiumMember member;
+
+        member.status = 0;
+
+        member.name = trim_all_ws(value["name"].getValStr()).substr(0, CConsortiumMember::MAX_CONSORTIUM_MEMBERS_STRING_LENGTH);
```

Comment:
Maybe enforce a minimum length here as well. "name" could either not be set or set blank.

Reply:
I left it optional as maybe there could be a member set without a name. Id is necessary and will be used as unique. This will all be set by foundation so didn't do fool proof validation.